Shared/src/com/pacoapp/paco/shared/util/QueryPreprocessor.java

```diff
@@ -103,13 +106,15 @@
                                SelectItemVisitor, OrderByVisitor {
 
   private Map<String, Class> validColumnNamesDataTypeInDb;
+  private Map<String, Long> dateParamWithLong = Maps.newHashMap();
   private List<String> requestedDateColumns;
   private String invalidColumnName;
   private String invalidDataType;
   private boolean containsExperimentIdClause;
   private boolean containsWhoClause;
+  private static final SimpleDateFormat sdfLocal = TimeUtil.localFormatter;
```

Comment:
Please rename to something like localDateFormatter (I know its pretty long but it is clearer :-) ) 
Or, given the convention of df that we have, it could be dfLocal to be shorter.